lib/modules/dosomething/dosomething_api/resources/kudos_resource.inc

```diff
@@ -207,5 +228,21 @@ function _kudos_resource_create($reportback_item_id, $term_ids, $northstar_id) {
  * @return string
  */
 function _kudos_resource_delete($kudos_id) {
+  if (variable_get('rogue_collection', FALSE)) {
+    global $user;
+    $northstar_user = dosomething_northstar_get_user($user->uid, 'drupal_id');
+
+    $data = [
+      'northstar_id' => $northstar_user->id,
+      'post_id' => $kudos_id,
+    ];
+
+    $response = dosomething_rogue_client()->postReaction($data);
```

Comment:
do we need to write logic to handle errors if a request doesn't make it to Rogue?